src/ParallelAlgorithms/Actions/InitializeItems.hpp

```diff
@@ -0,0 +1,75 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <optional>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Parallel/AlgorithmExecution.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace Parallel {
+template <typename Metavariables>
+class GlobalCache;
+}  // namespace Parallel
+/// \endcond
+
+namespace Initialization {
+namespace Actions {
+/*!
+ * \ingroup ActionsGroup
+ * \brief Mutate DataBox items by calling db::mutate_apply on each Mutator in
+ *  the order they are specified
+ *
+ * There's a specialization for `InitializeItems<tmpl::list<Mutators...>>` that
+ * can also be used if a `tmpl::list` is available.
+ *
+ * \details In addition to the requirements specified by db::mutate_apply, each
+ * Mutator must define the type aliases of this action.
```

Comment:
So currently, actions don't have to define a `*_global_cache_tags` alias, it'll be treated as empty if it doesn't exist. Just to be clear, we aren't having that option for Mutators. We are requiring every Mutator to have all these aliases even if they are empty? Also these aliases don't remove duplicates. Should we do that?

Reply:
Yes, my plan is to require them.  Removing duplicates here is probably a good thing to do as well

Reply:
Ok sounds good. Then yeah can you remove duplicates in all these lists and add some tests for that?

Reply:
I agree with Kyle :)